Templates/TimerTrigger-Powershell/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "defaultFunctionName": "TimerTriggerPowershell",
+    "description": "A Powershell function that will be run on a specified schedule",
+    "name": "TimerTrigger - Powershell",
+    "language": "Powershell",
+    "category": [
+        "Experimental",
+        "Data Processing"
```

Comment:
It's not really data processing, right? I see the other Timer samples are in the same category, but they're also in Core. Let's at least make sure it's consistent across all the Timer templates.